src/main/resources/public/js/components/leftPane.js

```diff
@@ -0,0 +1,168 @@
+
+        //     if (mountedApp.updateRecList) {
+        //         let i = 0;
+        //         while (i < this.recipeList.length) {
+        //             if (this.recipeList[i] !== mountedApp.curRec) {
+        //                 this.recipeList.splice(i, 1);
+        //                 this.recipeList.splice(i, 0, mountedApp.curRec);
+        //                 mountedApp.updateRecList = false;
+        //                 break;
+        //             }
+        //             i += 1;
+        //         }
+        //     }
+        //     mountedApp.curRec.ingredients.forEach((ingredient, idx) => {
+        //         const line = this.line(ingredient);
+        //         //        /// /
+        //         //        //        $('#recipe-details').append(`<li id="ing-edit${idx}">${getIngDiv(idx)}</li>`);
+        //         //
+        //         //        //        $(`#ing-edit${idx}`).hide();
+        //     });
+        //     //            $('button.edit-recipes').prop('disabled', false);
+        //     //            $('li > div > form > button.cancel').click(function () {
+        //     //              $('button.edit-recipes').prop('disabled', false);
+        //     //              ingCancel($(this).data('idx'));
+        //     //            });
+        //     //            $('li > div > form > button.apply').click(function () {
+        //     //              ingApply($(this).data('idx'));
+        //     //            });
+        //     //            $('button.openModal').click(function () {
+        //     //              currentIdx = $(this).data('idx');
+        //     //            });
+        //     //            $('input.inputIngInfo').keypress(function (event) {
+        //     //              const idx = $(this).data('idx');
+        //     //              if ($(`#quantity${idx}`).val() !== ''
+        //     //                && $(`#name${idx}`).val() !== ''
+        //     //                && event.keyCode === 13) {
+        //     //                $(`#add-ing${idx}`).click();
+        //     //              }
+        //     //            });
+        //     //            $('*').click(() => {
+        //     //              $('#error-message').hide();
+        //     //            });
+        //     //            $('input.inputIngInfo').focus(() => {
+        //     //              $('#error-message').hide();
+        //     //            });
+        //     console.log('all done! with the success!');
+        // },
+    },
+    /* html */
+    template: `
+    <div>
+        <div>
+            <strong class="text-primary">
+                <h2>Recipes:</h2>
+            </strong>
+            <button class="btn text-primary" data-placement="right" data-toggle="tooltip" id="plus-rec" title="Add recipe"
+                @click="showAddRecDiv">+</button>
+        </div>
+        <hr>
+        <div>
+            <!--    <div data-offset="0" data-spy="scroll" data-target="#recipe-list">-->
```

Comment:
It's ok to leave this commented out if you plan to bring it back in the future; but if not, just remove it.

Reply:
Done; I deleted it, and others like it.